src/components/Common/common.css

```diff
@@ -18,6 +18,17 @@ select {
     border-spacing: 0;
 }
 
+.lil-guitar {
```

Comment:
IMO guitar or any bg shud not be inside littlegiant .... @shivaprasadks plz suggest

Reply:
removed this and replaced with positioning, although had to add max width config class